src/index.ts

```diff
@@ -13,17 +13,22 @@ export async function run(): Promise<void> {
     const buildahPath = await io.which("buildah", true);
     const cli: BuildahCli = new BuildahCli(buildahPath);
 
+    const DEFAULT_TAG = "latest";
     const workspace = process.env.GITHUB_WORKSPACE || process.cwd();
     const dockerFiles = getInputList(Inputs.DOCKERFILES);
     const image = core.getInput(Inputs.IMAGE, { required: true });
-    const tags = core.getInput(Inputs.TAGS) || "latest";
+    const tags = core.getInput(Inputs.TAGS) || DEFAULT_TAG;
```

Comment:
hold on, doesn't this mean tagsList will always have at least one item in it?

Reply:
Ohh yes, I should add default tag to array after this.